openfl/component/aggregator/aggregator.py

```diff
@@ -243,7 +252,8 @@ def _time_to_quit(self):
             return True
         return False
 
-    def get_tasks(self, collaborator_name):
+    def get_tasks(self,
+                  collaborator_name: str) -> Tuple[Optional[List[str]], Optional[int], int, bool]:
```

Comment:
Why is the second argument returned optional?
```suggestion
                  collaborator_name: str) -> Tuple[Optional[List[str]], int, int, bool]:
```

And according to 295 line, returned tasks could be not only `None` or `List[str]`